indico/htdocs/sass/modules/_attachments.scss

```diff
@@ -332,18 +329,17 @@
 
 .material-list {
     .folder .attachment {
-        display: inline-block;
-        overflow: hidden;
+        @include ellipsis;
 
```

Comment:
I'll disagree, looking at `scss` code with line breaks after the `@include` and `@extend` feels less messy and more clean imo.